docs/docs/spec/metrics/metrics-prom.md

```diff
@@ -7,20 +7,38 @@ id: metrics-prom-spec
 
 The table below defines metrics which may be captured by AVS Nodes which expose metrics to Prometheus. AVSs may expose additional metrics however these should not use the `eigen_` prefix. 
 
+## Economics metrics
+
+| Name | Metric Type | Definition | Labels |
+|---|---|---|---|
+| `eigen_fees_earned_total` | Counter | The amount of fees earned in `<unit>` of underlying `<token>` in the `<strategy>` contract. This metric **SHOULD** be omitted while fees are yet to be implemented. | `token`, `unit`, `strategy` |
+| `eigen_slashing_status` | Gauge | Slashing status. The value **MUST** be 1 if the operator running `avs` has been slashed. This metric **SHOULD** not be implemented if there is no slashing in the target network (like in the upcoming testnet launch). | `avs` |
+| `eigen_registered_stakes` | Gauge | Operator stakes in AVS registry contract. Most commonly represents a weighted combination of delegated shares in the `DelegationManager` Eigenlayer contract. | `quorum` |
+
+:::note
+
+A definition of a strategy contract can be found in the [Glossary](/docs/glossary#strategy-contract).
+
+:::
+
+## Performance metrics
+
 | Name | Metric Type | Definition | Labels |
 |---|---|---|---|
-| `eigen_fees_earned_total` | Counter | The amount of fees earned in `<token>` | `token` |
-| `eigen_slashing_incurred_total` | Counter | The amount of slashing incurred in `<token>` | `token` |
-| `eigen_balance_total` | Gauge | AVS Node total balance in `<token>` | `token` |
 | `eigen_performance_score` | Gauge | The performance metric is a score between 0 and 100 and each developer can define their own way of calculating the score. The score is calculated based on the performance of the AVS Node and the performance of the backing  services. |  |
-| `eigen_rpc_request_duration_seconds` | Histogram | Duration of json-rpc `<method>` in seconds from Ethereum Execution client `<client>` | `method`, `client`, `version` |
-| `eigen_rpc_request_total` | Counter | Total of json-rpc `<method>` requests from Ethereum Execution client `<client>` | `method`,`client`,`version` |
+
+## RPC metrics
+
+| Name | Metric Type | Definition | Labels |
+|---|---|---|---| 
+| `eigen_rpc_request_duration_seconds` | Histogram | Duration of json-rpc `<method>` in seconds from Ethereum Execution client `<client>` | `method`, `client_version` |
+| `eigen_rpc_request_total` | Counter | Total of json-rpc `<method>` requests from Ethereum Execution client `<client>` | `method`, `client_version` |
 
 ## Notation examples
 
-* `eigen_fees_earned_total{token="ETH"}`
-* `eigen_slashing_incurred_total{token="ETH"}`
-* `eigen_total_balance_total{token="ETH"}`
+* `eigen_fees_earned_total{token="rETH", unit="gwei", strategy="0x1BeE69b7dFFfA4E2d53C2a2Df135C388AD25dCD2"}`
+* `eigen_slashing_status{avs="eigenDA}`
+* `eigen_registered_stakes{quorum="ethLST"}`
```

Comment:
Sorry one more change needed here... realized while trying to implement this that quorums are only stored as a number in the contracts. Can we change this to
```
eigen_registered_stakes{quorumNumber=0, quorumName="ethLST"}
```
and make quorumName optional (basically the offchain code can give a name to its quorums, but the underlying truth is just the quorumNumber stored in the contract)

Reply:
Done [4d8b0c1](https://github.com/NethermindEth/el-node-spec/pull/58/commits/4d8b0c10138bcb4ffe3dce6bc6c667a6a3859750).

I put labels in `snake_case` instead because that's a convention in Prometheus.